cpp/tests/io/parquet_test.cpp

```diff
@@ -1190,14 +1245,16 @@ TEST_F(ParquetReaderTest, UserBounds)
     auto expected = create_random_fixed_table<int>(4, 4, false);
 
     auto filepath = temp_env->get_temp_filepath("ZeroRowsPastBounds.parquet");
-    cudf_io::write_parquet_args args{cudf_io::sink_info{filepath}, *expected};
+    cudf_io::write_parquet_args args =
+      cudf_io::write_parquet_args::build(cudf_io::sink_info{filepath}, *expected);
     cudf_io::write_parquet(args);
 
     // attempt to read more rows than there actually are
-    cudf_io::read_parquet_args read_args{cudf_io::source_info{filepath}};
-    read_args.skip_rows = 4;
-    read_args.num_rows  = 0;
-    auto result         = cudf_io::read_parquet(read_args);
+    cudf_io::read_parquet_args read_args =
+      cudf_io::read_parquet_args::build(cudf_io::source_info{filepath})
+        .with_skip_rows(4)
```

Comment:
Another verb in the parameter name. Not sure how best to approach these.

Reply:
How about use the member name itself in API, just `skip_rows(10)`.